config/database.travis.yml

```diff
@@ -0,0 +1,89 @@
+# PostgreSQL. Versions 9.1 and up are supported.
+#
+# Install the pg driver:
```

Comment:
this shouldn't be here. Did you rebase everything already?

Reply:
ohh.. I didn't see it. Yes, I rebase it.

Reply:
check that you shouldn't have this commit: `add travis config and database.travis.yml`